lib/feature_test.rb

```diff
@@ -0,0 +1,11 @@
+#require_relative "menu"
+#require_relative "show_menu"
+require_relative "order"
+require_relative "confirm_order"
+
+#m = Menu.new
+o = Order.new
```

Comment:
A feature test should be run the same way unit tests are - through rspec and with assertions. You're not asserting anything here so the only thing it's really testing is that the program runs without errors.

With feature tests, you should have one test per user story and an assertion that matches the outcome of the user story. For example, for the story:
```
As a customer
So that I can check if I want to order something
I would like to see a list of dishes with prices
```
I would expect to see a test like:

```
describe "Feature" do
  describe "Show menu"
    ... setup
    expect(app.show_menu).to eq("... expected menu output ...")
  end
end
```